app/assets/stylesheets/application.scss

```diff
@@ -56,7 +56,14 @@ $search-result: #008cba;
   .text-field {
     padding: 5px;
     height: 25px;
-    margin-bottom: 15px;
+    margin-bottom: 1px;
+  }
+
+  .error-explanation {
+    color: #ff0000;
```

Comment:
Color literals like `#ff0000` should only be used in variable declarations; they should be referred to via variable everywhere else.